internal/commands/config_registries.go

```diff
@@ -0,0 +1,173 @@
+package commands
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+
+	"github.com/buildpacks/pack/internal/config"
+	"github.com/buildpacks/pack/internal/slices"
+	"github.com/buildpacks/pack/internal/stringset"
+	"github.com/buildpacks/pack/internal/style"
+	"github.com/buildpacks/pack/logging"
+	"github.com/buildpacks/pack/registry"
+)
+
+var (
+	bpRegistryExplanation = "A Buildpack Registry is a (still experimental) place to publish, store, and discover buildpacks. "
+)
+
+var (
+	setDefault   bool
+	registryType string
+)
+
+func ConfigRegistries(logger logging.Logger, cfg config.Config, cfgPath string) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:     "registries",
+		Aliases: []string{"registry", "registreis"},
```

Comment:
I'm not sure whether or not to include `registreis` as an alias. I figure that since it's a long word, and [i before e is confusing](https://www.dailymail.co.uk/news/article-1194246/Forget-e-rule-teachers-told--just-confusing.html), it could be helpful to plan for it.